topology/src/main/java/indexingTopology/util/shape/CheckInCircle.java

```diff
@@ -0,0 +1,26 @@
+package indexingTopology.util.shape;
+
+/**
+ * Create by zelin on 17-11-29
+ **/
+public class CheckInCircle implements CheckContain {
+
+    private double longitude;
+    private double latitude;
+    private double radius;
+    public CheckInCircle() {
```

Comment:
You need to take the input arguments of the constructor to initialize the member variable, rather than setting them by hard-code.